src/Sulu/Bundle/WebsiteBundle/Controller/SitemapController.php

```diff
@@ -25,59 +22,137 @@
 class SitemapController extends WebsiteController
 {
     /**
-     * Returns a rendered xmlsitemap.
+     * Render sitemap-index of all available sitemap.xml files.
+     * If only one provider exists this provider will be rendered directly.
```

Comment:
This controller is not called if somebody has dumped a `sitemap.xml` file, right? Then I would add a comment here explaining that.